content/en/tracing/profiler/getting_started.md

```diff
@@ -256,6 +256,80 @@ The Datadog Profiler requires Go 1.12+. To begin profiling applications:
 [6]: /tracing/visualization/#services
 [7]: /tracing/guide/setting_primary_tags_to_scope/#environment
 {{< /programming-lang >}}
+{{< programming-lang lang="ruby" >}}
+
+<div class="alert alert-warning">
+Datadog Ruby Profiler is currently in public beta. We strongly advise evaluating the profiler in a non-sensitive environment before deploying in production.
+</div>
+
+The Datadog Profiler requires MRI Ruby 2.1+. **Wall time profiling is available for users on every platform (including macOS and Windows), but CPU time profiles are currently only available on Linux platforms**. To begin profiling applications:
+
+1. If you are already using Datadog, upgrade your agent to version [7.20.2][1]+ or [6.20.2][1]+.
+
+2. Add the `ddtrace` and `google-protobuf` gems to your `Gemfile` or `gems.rb` file:
+
+    ```ruby
+    gem 'ddtrace'
+    gem 'google-protobuf', '~> 3.0'
+    ```
+
+    **Note**: Profiler is available in the `ddtrace` library for versions 0.49+.
+
+2. Install the gems with `bundle install`
+
+3. You can auto-enable the profiler with environment variables:
+
+    ```shell
+    DD_PROFILING_ENABLED=true
+    DD_ENV=prod
+    DD_SERVICE=my-web-app
+    DD_VERSION=1.0.3
+    ```
+
+    or in code:
+
+    ```ruby
+    Datadog.configure do |c|
+
+      # This will enable the profiler
+      c.profiling.enabled = true
+      c.profiling.enabled = true
```

Comment:
Duplicated :)

```suggestion
    ```ruby
    Datadog.configure do |c|
      # This will enable the profiler
      c.profiling.enabled = true
```